engine/src/main/scala/cromwell/server/CromwellAkkaLogFilter.scala

```diff
@@ -0,0 +1,20 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import akka.event.EventStream
+import akka.event.slf4j.Slf4jLoggingFilter
+
+class CromwellAkkaLogFilter(settings: ActorSystem.Settings, eventStream: EventStream) extends Slf4jLoggingFilter(settings, eventStream) {
+  override def isErrorEnabled(logClass: Class[_], logSource: String) = {
+    /*
+     * This might filter out too much but it's the finest granularity we have hear
```

Comment:
here

Reply:
are you aware of anything specifically being filtered other than the bad stuff?

Reply:
Not specifically no